src/ucs/arch/cpu.h

```diff
@@ -35,6 +36,7 @@ typedef enum ucs_cpu_model {
     UCS_CPU_MODEL_AMD_ROME,
     UCS_CPU_MODEL_AMD_MILAN,
     UCS_CPU_MODEL_AMD_GENOA,
+    UCS_CPU_MODEL_LOONGARCH64,
```

Comment:
Shouldn't these new enum values normally be appended to the end of the list, to prevent changing the values of existing names?